-1

Tested at: 0bc2dcdf9b2d4eac714b2e28f30ff510cc728b46

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15347/14359/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-08-04-1100/src/Geometry/EcalCommonData/src/EcalBarrelNumberingScheme.cc 
In file included from /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-08-04-1100/src/Geometry/EcalCommonData/interface/DDEcalEndcapTrap.h:8:0,
                 from /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-08-04-1100/src/Geometry/EcalCommonData/src/DDEcalEndcapTrap.cc:1:
/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-08-04-1100/src/DetectorDescription/Base/interface/DDutils.h:1:1: warning: null character(s) ignored
In file included from /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-08-04-1100/src/Geometry/CSCGeometryBuilder/src/CSCGeometryBuilderFromDDD.cc:2:0:
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-08-04-1100/src/Geometry/CSCGeometryBuilder/src/CSCGeometryBuilder.h:32:17: error: 'RecoIdealGeometry' does not name a type
         , const RecoIdealGeometry& rig
                 ^
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-08-04-1100/src/Geometry/CSCGeometryBuilder/src/CSCGeometryBuilder.h:33:17: error: 'CSCRecoDigiParameters' does not name a type
         , const CSCRecoDigiParameters& cscpars ) ;
                 ^
</pre>

--- a/Fireworks/Geometry/src/TGeoFromDddService.cc
+++ b/Fireworks/Geometry/src/TGeoFromDddService.cc
@@ -316,7 +316,7 @@ TGeoFromDddService::createShape(const std::string& iName,
                                     params[2]/cm,
                                     params[0]/cm,
                                     params[3]/deg,
-                                    params[4]/deg);
+                                    params[3]/deg + params[4]/deg);
 	    break;
 	 case ddcuttubs:
 	    //Order in params is  zhalf,rIn,rOut,startPhi,deltaPhi
@@ -326,7 +326,7 @@ TGeoFromDddService::createShape(const std::string& iName,
 				 params[2]/cm,
 				 params[0]/cm,
 				 params[3]/deg,
-				 params[4]/deg,
+				 params[3]/deg + params[4]/deg,
 				 params[5],params[6],params[7],
 				 params[8],params[9],params[10]);
 	    break;

--- a/Fireworks/Geometry/src/TGeoMgrFromDdd.cc
+++ b/Fireworks/Geometry/src/TGeoMgrFromDdd.cc
@@ -271,7 +271,7 @@ TGeoMgrFromDdd::createShape(const std::string& iName,
 				 params[2]/cm,
 				 params[0]/cm,
 				 params[3]/deg,
-				 params[4]/deg,
+				 params[3]/deg + params[4]/deg,
 				 params[5],params[6],params[7],
 				 params[8],params[9],params[10]);
 	    break;

